icon/icmodule/revision.go

```diff
@@ -104,7 +104,8 @@ const (
 	// Unused
 	// RevisionJavaFixMapValues = Revision20
 
-	RevisionBTP2 = Revision21
+	RevisionBTP2            = Revision21
+	RevisionUpdatePRepStats = Revision21
```

Comment:
We cannot use Revision21 for this feature since Berlin net has already been updated to that rev.

Reply:
Ok. I'll change start revision from 21 to 22.